src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carCost = 40;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+  const maxLine = 7;
+  const minLine = 3;
+
+  if (days >= maxLine) {
+    return carCost * days - maxDiscount;
+  }
+
+  if (days < maxLine && days >= minLine) {
```

Comment:
you don't need `days < maxLine` check here, if `days >= maxLine` will be false then `days < maxLine` always will be true

Reply:
oh, right